RFS/src/main/java/com/rfs/models/GlobalMetadata.java

```diff
@@ -23,6 +23,8 @@ public interface GlobalMetadata {
     */
     public ObjectNode toObjectNode();
 
+    public ObjectNode getTemplates();
```

Comment:
Wow - I'm honestly surprised this wasn't in here already.  I wonder now why I didn't add it...